02.calendar/cal.rb

```diff
@@ -0,0 +1,33 @@
+require 'date'
+require 'optparse'
+
+opt = OptionParser.new
+
+params = {}
+
+opt.on('-y [VAL]') {|v| v }
+opt.on('-m [VAL]') {|v| v }
+
+opt.parse!(ARGV, into: params)
+
+if params[:y].nil?
+  year = Date.today.year
+  month = params[:m].nil? ? Date.today.month : params[:m].to_i
+else
+  year = params[:y].to_i
+  month = params[:m].to_i
+end
+
+month_beginning = Date.new(year, month, 1)
+month_end = Date.new(year, month, -1)
+
+puts "#{month}月 #{year}".center(21) 
+puts " 日 月 火 水 木 金 土"
+
+default_output_width = 3
```

Comment:
これはいつでも同じ値になるので、変数よりも定数にした方が良さそうですね。
また、名前も短くしちゃって問題ないかもしれません〜。

```suggestion
WIDTH = 3
```

Reply:
ご確認いただき、ありがとうございます！ご指摘通りに修正しました。
ご指摘していただいて、たしかに定数の方がよかったことに気づきました。